patches/minecraft/net/minecraft/entity/monster/EntityZombieVillager.java.patch

```diff
@@ -8,7 +8,28 @@
      }
  
      public int func_190736_dl()
-@@ -85,6 +86,21 @@
+@@ -64,6 +65,7 @@
+     {
```

Comment:
None of the changes in this file should be needed, they were only hacks because the old system mixed in profession and normal villager types.
Now that that is no longer the case, we don't have to special case the profession string.
What MAY need to happen is us going in and removing any explicit limitations to the number 6, the number of vanilla professions.